### PR TITLE
PricePerShare and creation/update timestamp for signals

### PR DESCRIPTION
**Pull request by juanmardefago**

### CODE REVIEW

**Review by nenadjaja — APPROVED** [review]
Looks good to me :) I can actually understand subgraph code way more than I thought hahah

**Comment by nenadjaja** [comment]
@juanmardefago  can you merge this pls? I forgot if a person who approves the PR merges it or the person who created the PR? (on the product eng team the person who created the PR merges it after approval) :)

**Comment by juanmardefago** [comment]
> @juanmardefago can you merge this pls? I forgot if a person who approves the PR merges it or the person who created the PR? (on the product eng team the person who created the PR merges it after approval) :)

I didn't merge it just after the approval because Dave said he'd take a look, but tbh it's pretty simple and if we find anything buggy when testing we can just fix it, after all it's only merged to staging

**Pull request merged**
